src/components/blog/post/body.module.css

```diff
@@ -0,0 +1,358 @@
+.root {
+  font-family: "Acta", serif;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 28px;
+}
+
+/* Limit block elements w/ paddings */
+
+.root blockquote,
+.root h2,
+.root h3,
+.root h4,
+.root h5,
+.root h6,
+.root ol,
+.root p,
+.root ul {
+  max-width: 618px;
+  padding: 0 20px;
+}
+
+.root h2,
+.root h3,
+.root h4,
+.root h5,
+.root h6,
+.root ol,
+.root p,
+.root ul {
+  margin-right: auto;
+  margin-left: auto;
+}
+
+/* Anchors */
+
+.root a,
+.root a:hover,
+.root a:link,
+.root a:visited {
+  color: #2421ab;
+
+  font-weight: 700;
+  text-decoration: none;
+}
+
+.root a:hover {
+  opacity: 0.5;
+
+  transition: opacity 0.2s;
+}
+
+/* Code samples */
+
+.root code {
+  font-family: source-code-pro, monospace;
+  font-weight: 400;
+}
+
+/* Headings */
+
+.root h2,
+.root h3,
+.root h4,
+.root h5,
+.root h6 {
+  font-family: colfax-web, sans-serif;
+  font-weight: 500;
+}
+
+.root h2 {
+  margin-top: 84px;
+  margin-bottom: 20px;
+
+  font-size: 20px;
+  line-height: 28px;
+}
+
+.root h3 {
+  margin-top: 40px;
+  margin-bottom: 20px;
+
+  font-size: 20px;
+  line-height: 28px;
+}
+
+/* Paragraph */
+
+.root > p:first-child::first-letter {
+  float: left;
+  padding-right: 4px;
+  margin-top: 0;
+
+  font-family: "Acta Display", serif;
+  font-size: 56px;
+  line-height: 56px;
+}
+
+.root p:not(:first-child) {
+  margin-top: 20px;
+}
+
+.root p:not(:last-child) {
+  margin-bottom: 28px;
+}
+
+/* Images (in paragraphs) */
+
+.root p > img {
```

Comment:
If you want, there is a plugin that unwraps images out of the `p` tags that the `remark` engine creates. I think accesibility wise that is a bit stupid, to have an image inside a paragraph. https://www.gatsbyjs.org/packages/gatsby-remark-unwrap-images/